Cargo.toml

```diff
@@ -58,6 +58,7 @@ include = [
     "crypto/fipsmodule/bn/asm/x86-mont.pl",
     "crypto/fipsmodule/bn/asm/x86_64-mont.pl",
     "crypto/fipsmodule/bn/asm/x86_64-mont5.pl",
+    "crypto/fipsmodule/bn/asm/mips-mont.pl",
```

Comment:
I understand why you're trying to do it this way, but this is exactly what I'm hoping to avoid. Let's find a way to do it without bringing in any MIPS assembly code. It's just not realistic to maintain the MIPS assembly code given current constraints. I'd rather concentrate on improving the C (in the near future, Rust) code instead.

Reply:
I'm mostly reverse-engineering the error messages which complained about `bn_mul_mont` not existing, and I found this file defines it. I'm ok with removing it if I can figure out another way to overcome the error message.